Updated the documentation for the four new emissions api endpoints. (#219)
Trackunit/developer-hub: changelogs/emissions-api-add-site-assets-monthly.md

---
title: Emissions API Site Assets Update
type: added
---

Introducing four new endpoints to the Emissions API that allow a breakdown of assets emissions values by site summed up on a monthly basis.

> These endpoints provide emissions values for the period they have been on the site, this is different from the existing monthly endpoints that provide emissions values for the period they have been in use.

Reviewer: ```suggestion
These endpoints provide emissions values for the period they have been on the site.  These endpoints are different from the existing monthly endpoints that provide emissions values for the period they have been in use.
```
